### ISSUE TITLE
Failed to add HDB / IPTorrents

### ISSUE THREAD

**Issue by LooseSeal2** [Radarr/Radarr#3713]
**Describe the bug**
IPTorrents and HDB APIs were updated. Based on where it's failing, the response model probably needs updates.

 I do have 2FA enabled on my hdb account. Sonarr Phantom works fine with both but Radarr Aphrodite doesn't. 

**To Reproduce**
1. Go to Add an indexer
2. Click on HDB
3. enter info, click test
4. See error icon, no msg why

repeat for IP torrents

**Expected behavior**
Works.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Docker, Ubuntu
- Mono: 5.20.1.34
 - Version: 2.0.0.1422

**Debug Logs**
Relevant log:
```
System.NullReferenceException: Object reference not set to an instance of an object
  at Radarr.Http.ClientSchema.SchemaBuilder+<>c.<GetValueConverter>b__8_6 (System.Object fieldValue) [0x00000] in C:\projects\radarr-usby1\src\Radarr.Http\ClientSchema\SchemaBuilder.cs:189 
  at Radarr.Http.ClientSchema.SchemaBuilder+<>c__DisplayClass5_2.<GetFieldMapping>b__2 (System.Object t, System.Object v) [0x00000] in C:\projects\radarr-usby1\src\Radarr.Http\ClientSchema\SchemaBuilder.cs:124 
  at Radarr.Http.ClientSchema.SchemaBuilder.ReadFromSchema (System.Collections.Generic.List`1[T] fields, System.Type targetType) [0x00096] in C:\projects\radarr-usby1\src\Radarr.Http\ClientSchema\SchemaBuilder.cs:49 
  at Radarr.Api.V2.ProviderResourceMapper`2[TProviderResource,TProviderDefinition].ToModel (TProviderResource resource) [0x000c8] in C:\projects\radarr-usby1\src\Radarr.Api.V2\ProviderResource.cs:65 
  at Radarr.Api.V2.Indexers.IndexerResourceMapper.ToModel (Radarr.Api.V2.Indexers.IndexerResource resource) [0x00005] in C:\projects\radarr-usby1\src\Radarr.Api.V2\Indexers\IndexerResource.cs:37 
  at Radarr.Api.V2.ProviderModuleBase`3[TProviderResource,TProvider,TProviderDefinition].GetDefinition (TProviderResource providerResource, System.Boolean includeWarnings, System.Boolean validate) [0x00000] in C:\projects\radarr-usby1\src\Radarr.Api.V2\ProviderModuleBase.cs:100 
  at Radarr.Api.V2.ProviderModuleBase`3[TProviderResource,TProvider,TProviderDefinition].Test (TProviderResource providerResource) [0x00000] in C:\projects\radarr-usby1\src\Radarr.Api.V2\ProviderModuleBase.cs:141 
  at Radarr.Api.V2.ProviderModuleBase`3[TProviderResource,TProvider,TProviderDefinition].<.ctor>b__2_1 (System.Object x) [0x00000] in C:\projects\radarr-usby1\src\Radarr.Api.V2\ProviderModuleBase.cs:30 
  at (wrapper dynamic-method) System.Object.CallSite.Target(System.Runtime.CompilerServices.Closure,System.Runtime.CompilerServices.CallSite,System.Func`2<object, object>,object)
  at Nancy.Routing.Route+<>c__DisplayClass4.<Wrap>b__3

```

[ipt-radarr.debug.txt](https://github.com/Radarr/Radarr/files/3532431/ipt-radarr.debug.txt)
[hdb-radarr.debug.txt](https://github.com/Radarr/Radarr/files/3532432/radarr.debug.txt)

**Comment by HLFH** [comment]
Same issue with TorrentLeech. You cannot add indexers with Radarr Aphrodite.

**Comment by fryfrog** [comment]
In the mean time, Jackett is *much* faster at updating indexers and I much prefer using it for any of them that don't have some special benefit from being native (like PTP and BTN).

**Comment by LooseSeal2** [comment]
Unable to add jackett, same error.

**Comment by Qstick** [comment]
Fixed f43210d3d
